charts/pulsar/templates/dashboard-ingress.yaml

```diff
@@ -36,6 +36,9 @@ metadata:
   name: "{{ template "pulsar.fullname" . }}-{{ .Values.dashboard.component }}"
   namespace: {{ template "pulsar.namespace" . }}
 spec:
+  {{- with .Values.dashboard.ingress.ingressClassName }}
```

Comment:
I haven't used [with](https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with) before. That is very handy.